content/400-reference/200-api-reference/300-environment-variables-reference.mdx

```diff
@@ -66,6 +62,10 @@ const shouldHide = process.env.PRISMA_HIDE_UPDATE_MESSAGE
 
 Disables all CLI warnings generated by `logger.warn`.
 
+### <inlinecode>PRISMA_GENERATE_DATAPROXY</inlinecode>
+
+To configure `prisma generate` to [generate a Data Proxy Client](/concepts/data-platform/data-proxy) on any execution, even when `--data-proxy` is not explicitly set for the CLI command, set the value of this environment value to a truthy value (e.g. `PRISMA_GENERATE_DATAPROXY=true`).
```

Comment:
I'll make another strong stand against using "Data Proxy Client" because it sounds like a new component. Also, when we spell it out, it helps readers understand the relationship between Client and the data proxy.

Reply:
The full spelling out would be "Prisma Client for Data Proxy" that you did not like in another location. 

What do you suggest to do here?

Reply:
Changing to the long one for now:
```suggestion
To configure `prisma generate` to [generate a Prisma Client for Data Proxy](/concepts/data-platform/data-proxy) on any execution, even when `--data-proxy` is not explicitly set for the CLI command, set the value of this environment value to a truthy value (e.g. `PRISMA_GENERATE_DATAPROXY=true`).
```

Reply:
Persisted in https://github.com/prisma/docs/issues/3285